superset/models/core.py

```diff
@@ -693,7 +749,7 @@ class Database(Model, AuditMixinNullable, ImportMixin):
     impersonate_user = Column(Boolean, default=False)
     export_fields = ('database_name', 'sqlalchemy_uri', 'cache_timeout',
                      'expose_in_sqllab', 'allow_run_async',
-                     'allow_ctas', 'allow_csv_upload', 'extra')
+                     'allow_ctas', 'allow_csv_upload', 'extra', 'uuid')
```

Comment:
as mentioned above, uuid should be implicit

Reply:
Although it is a part of `ImportExportMixin`, I had thought this was needed. I'll check that it works without it.

Reply:
Fixed.